build.xml

```diff
@@ -9,39 +9,50 @@
   <property name="skip.tests" value="true"/>
    -->
 
-	<property name="version" value="0.0.0" />
+	<exec executable="git" outputproperty="git.revision">
+		<arg value="describe" />
+		<arg value="--always" />
+	</exec>
+	<property name="version" value="0.0.0-${git.revision}" />
```

Comment:
Create as a todo, so that the first three version parts (M.m.b) being grabbed from a version file which is managed by the user to decide when to uptick things (e.g. when adding breaking API changes).  Look at core umple for an example